src/components/InviteParticipants/InviteForm.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import { Formik } from 'formik'
+import { Form as BaseForm, Button } from 'react-bootstrap'
+import styled from 'styled-components'
+import { useTranslation } from 'react-i18next'
+import classNames from 'classnames'
+import Field from '../Form/Field'
+import CallError, { Error } from '../Form/CallError'
+
+const BootstrapForm = styled(BaseForm)`
+    width: 100%;
+`
+
+const FormFields = styled.div`
+    & > * {
+        margin: 0 0 ${({ theme }) => theme.spacing.XXL};
+    }
+
+    &:first-child {
+        margin-top: ${({ theme }) => theme.spacing.XXL};
+    }
+`
+
+const SubmitError = styled.p`
+    color: ${({ theme }) => theme.color.red};
+`
+
+const FormCardHeader = styled.div`
+    & p {
+        &:first-child {
+            text-align: center;
+            font-weight: bold;
+            font-size: ${({ theme }) => theme.font.XL};
+            color: ${({ theme }) => theme.color.orange};
+        }
+    }
+`
+
+const FormSubmit = styled.div`
+    margin-bottom: ${({ theme }) => theme.spacing.XXL};
+    text-align: center;
+    Button {
+        margin: ${({ theme }) => theme.spacing.XXS};
+    }
+`
+
+export default function InviteForm({ onSubmit }) {
+    const { t } = useTranslation('add-participants-form')
+    const error: Error = {
+        code: '500',
+        message: t('errors.inviteParticipants'),
+    }
+
+    const initialValues = {
+        participantAddresses: [],
+    }
+
+    const onValidate = async (values) => {
+        // TODO add validation logic
```

Comment:
using YUP, + create a Field directly linked to Formik to handle error msg display

Reply:
- validation logic will be implemented once the participants invite input field will be replaced with the more complex component also handling phone numbers and groups
- I reused `CallError` and only set a generic error message in case the network operation fails

Reply:
Created #328 for this one